modmail/database/models/messages.py

```diff
@@ -0,0 +1,26 @@
+from typing import TYPE_CHECKING
+
+from tortoise import fields
+from tortoise.models import Model
+
+
+if TYPE_CHECKING:
+    from .attachments import Attachments
+    from .embeds import Embeds
+    from .emojis import Emojis
+    from .stickers import Stickers
+
+
+class Messages(Model):
```

Comment:
channel field.

Reply:
Why do messages need a channel field? It is already referencing to the modmail ticket it was sent in.